### ISSUE TITLE
[studio-ui] Can't create components using the child-content datasource

### ISSUE THREAD

**Issue by jdrossl** [craftercms/craftercms#3541]
## Describe the bug
When trying to create a component in a `item-selector` using the `child-content` the form is not shown.

## To Reproduce
1. Create and empty site
2. Create a content-type for a component
3. Add a item-selector & child-content to the home page content-type
4. Edit the home page, try to create a new component from the item-selector control
5. Nothing happens

## Expected behavior
A form should be opened to create the new component

## Logs
In the browser console:
```
Uncaught TypeError: Cannot read property 'indexOf' of undefined
    at Object.processPathsForMacros (common-api.js?version=057c6ae0b249b1f4c256ef8698a82e5e714d8164:2988)
    at CStudioForms.Datasources.ChildContent.processPathsForMacros (forms-engine.js?version=057c6ae0b249b1f4c256ef8698a82e5e714d8164:129)
    at CStudioForms.Datasources.ChildContent.createElementAction (<anonymous>:90:23)
    at HTMLDivElement.<anonymous> (<anonymous>:171:27)
    at HTMLDivElement.P (utilities.js?version=057c6ae0b249b1f4c256ef8698a82e5e714d8164:11)
processPathsForMacros @ common-api.js?version=057c6ae0b249b1f4c256ef8698a82e5e714d8164:2988
processPathsForMacros @ forms-engine.js?version=057c6ae0b249b1f4c256ef8698a82e5e714d8164:129
createElementAction @ VM4131:90
(anonymous) @ VM4131:171
P @ utilities.js?version=057c6ae0b249b1f4c256ef8698a82e5e714d8164:11
```

### Version
3.1.3

**Comment by sumerjabri** [comment]
Already fixed in `develop` 3.1.4-SNAPSHOT

**Comment by russdanner** [comment]
A work around to this issue:
Modify the form definition (form-defintition.xml) for the content type and add the following property to the data source that has this issue:
```
<property>
	<name>repoPath</name>
	<value>/site/components/features/</value>
	<type>undefined</type>
</property>
```

This bug applies to new data sources only.  Existing data sources will work but the value repository path property value will not be present in the content type editor.
